sonar-plugin-api/src/main/java/org/sonar/api/issue/ProjectIssues.java

```diff
@@ -19,18 +19,20 @@
  */
 package org.sonar.api.issue;
 
-import org.sonar.api.batch.ScannerSide;
 import org.sonar.api.batch.InstantiationStrategy;
+import org.sonar.api.batch.ScannerSide;
 
 import static org.sonar.api.batch.InstantiationStrategy.PER_BATCH;
 
 /**
  * Used by batch components to get all project issues.
  *
  * @since 4.0
+ * @deprecated since 7.0 preview mode is deprecated for a while
  */
 @InstantiationStrategy(PER_BATCH)
 @ScannerSide
+@Deprecated
 public interface ProjectIssues {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)